src/Generator.php

```diff
@@ -156,6 +157,14 @@ public function createClassNameDetails(string $name, string $namespacePrefix, st
         return new ClassNameDetails($className, $fullNamespacePrefix, $suffix);
     }
 
+    /**
+     * @experimental The method signature may change, moved, or be removed.
```

Comment:
We should keep this experimental until we've implemented with a few different makers as the `Generator::class` is not `internal` or `final`

Reply:
that didnt last long